src/main.rs

```diff
@@ -1,99 +1,108 @@
-extern crate hyper;
-extern crate hyper_rustls;
-extern crate yup_oauth2 as oauth2;
+//extern crate datastore_entity;
 extern crate google_datastore1 as datastore1;
```

Comment:
Rust 2018 - not required to use `extern crate` except some rare cases. Just `use google_datastore1 as datastore1`